package/rhel/postinstall.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+if [ "$1" -eq 1 ] && [ -x "/usr/lib/systemd/systemd-update-helper" ]; then
+	# Initial installation
+	/usr/lib/systemd/systemd-update-helper install-system-units frankenphp.service || :
+fi
+
+if [ -x /usr/sbin/getsebool ]; then
+	# connect to ACME endpoint to request certificates
```

Comment:
```suggestion
	# Connect to ACME endpoint to request certificates
```

Same for all comments for consistency